requirements.lock

```diff
@@ -1,6 +0,0 @@
-dependencies:
-- name: postgresql
-  repository: https://charts.bitnami.com/bitnami
-  version: 9.2.0
-digest: sha256:5082c9ee31559622b64a7391ad9d2f8cf8488a0d34ff03fb1600dfac5f3d89e2
-generated: "2020-08-28T15:21:20.204476-04:00"
```

Comment:
The [helm documentation](https://helm.sh/docs/faq/#consolidation-of-requirementsyaml-into-chartyaml) mentioned that `requirements.lock` was consolidated into `Chart.lock`, is that missing from this PR?

Reply:
Indeed. We definitely missed committing that.